backend/Dockerfile

```diff
@@ -1,23 +1,15 @@
 FROM golang:alpine as builder
 
-RUN apk add --no-cache curl git
-RUN curl -fsSL -o /bin/dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64 && chmod +x /bin/dep
-
-# Download and install the latest release of dep
-ADD https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64 /usr/bin/dep
-RUN chmod +x /usr/bin/dep
-
 WORKDIR /go/src/github.com/kubeflow/pipelines
 COPY . .
 
-# Needed for github.com/mattn/go-sqlite3
```

Comment:
could you keep this comment

Reply:
sorry, accidently removed this line.

will add it back in next pr.